.github/workflows/reusable-publish-docs.yml

```diff
@@ -1,22 +1,28 @@
 name: Reusable Publish docs
 
+env:
+  BRANCH: main
+  ORIGIN: awslabs/aws-lambda-powertools-python
+
+
 on:
   workflow_call:
     inputs:
-      workflow_origin: # see https://github.com/awslabs/aws-lambda-powertools-python/issues/1349
+      version:
+        description: "Version to build and publish docs (1.28.0, develop)"
```

Comment:
Does the version number requires a `v` prefix or not? I.e. is it `v1.5.1` or `1.5.1`? Asking because up until now it was required in other workflows. 

If it is, can we update the description to remind the reader to add it?

Reply:
It was required because we took used tag and the `ref_name` with slicing the 'v' out:
```
RELEASE_VERSION=$(echo ${{ github.ref_name }} | sed 's/v//')
```

It's not necessary any more.